code/modules/client/preference_setup/loadout/items/suit.dm

```diff
@@ -542,3 +542,17 @@
 	display_name = "visegradi ponczo"
 	path = /obj/item/clothing/suit/storage/hooded/wintercoat/colorable/ponczo
 	flags = GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/suit/galatea_labcoat
```

Comment:
Put this in the labcoat selection with the rest, don't make singleton selections.

Reply:
Individual selections can't be origin-restricted, which is preferable for origin-specific items.

Reply:
I think it's fine as their own selection, not added to other labcoats.
This would be consistent with many other loadout items.

In the same way that konyanger jackets were not added to the generic jacket selection.
Or visegradii poncho is not in the generic poncho selection.
Or dominian suits are not in the generic suits selection.